docs/proto/proto-docs.md

```diff
@@ -238,6 +241,21 @@ Query provides defines the gRPC querier service
 
 
 
+<a name="osmosis.meshsecurity.v1beta1.ScheduledWork"></a>
+
+### ScheduledWork
+
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| `repeat` | [bool](#bool) |  |  |
+
```

Comment:
What is this? Can you add some more context here?

Reply:
Not sure about this. This was generated automatically when generated proto file.